_maps/map_files/Pahrump-Old/Pahrump-Sunset-Upper.dmm

```diff
@@ -995,7 +995,15 @@
 	},
 /turf/open/indestructible/ground/outside/woodalt,
 /area/f13/building)
-"er" = ()
+"er" = (
+/obj/structure/table,
+/obj/effect/decal/cleanable/dirt,
+/obj/structure/window/spawner/north,
+/obj/item/paper/crumpled/bloody/ruins/thederelict/unfinished{
+	info = "Why's the [ILLEGIBLE] Here?? They're screaming something about [ILLEGIBLE]... No.. this can't be.."
```

Comment:
byond considers [square brackets] in quotes to be replacement bits for a variable in the square bracket
ie: `"There are [number_of_apples] in this pile."`, where number_of_apples is a variable.

Which is why MDB is failing, because map files can't be changed during compile-time.

So, all you have to do is escape the leading square bracket in each set.
ie: `Why's the \[ILLEGIBLE] Here??`